scripts/includes.sh

```diff
@@ -83,6 +84,55 @@ function send_mail_content() {
     fi
 }
 
+########################################
+# Export variables from .env file.
+# Arguments:
+#     None
+# Outputs:
+#     variables with prefix 'DOTENV_'
+# Reference:
+#     https://gist.github.com/judy2k/7656bfe3b322d669ef75364a46327836
```

Comment:
Please replace with this: https://gist.github.com/judy2k/7656bfe3b322d669ef75364a46327836#gistcomment-3625311

I had to search that comment which is uncomforatble. This link jumps directly to that comment

Reply:
Thanks!